packages/react/src/generators/application/lib/add-e2e.ts

```diff
@@ -55,6 +55,7 @@ export async function addE2e(
         webServerCommand: `${getPackageManagerCommand().exec} nx serve ${
           options.name
         }`,
+        webServerAddress: 'localhost:4200',
```

Comment:
in my testing just `localhost:<port>` doesn't work. it needs the protocol as well. `http://localhost:<port>`
```suggestion
        webServerAddress: 'http://localhost:4200',
```

Reply:
changed